### ISSUE TITLE
pyhessio issues on CentOS6

### ISSUE THREAD

**Issue by dneise** [cta-sst-1m/digicampipe#227]
@Etienne12345 told me pyhessio does not work on SL6. 

We are unfortunately importing pyhessio by default in [`event_stream.py`](https://github.com/cta-sst-1m/digicampipe/blob/master/digicampipe/io/event_stream.py#L1)

I believe if pyhession is not installed we throw an exception, even if the user will never even try to read a hessio file.

It would be better to import this stuff only when really needed.  ctapipe is adopting the same strategy here: only import if really needed.

Should be easy to fix, just writing it here to not forget about it.

**Comment by calispac** [comment]
That sounds good enough. @jurysek do you have an opinion? since you are using pyhessio

**Comment by dneise** [comment]
What I have in mind should not affect @jurysek in any way. He will not realize that the import is only done, when he actually tries to read the first file instead of doing it on first import.

Also runtime should not be negatively affected.

**Comment by calispac** [comment]
What I realized on "baobab" is that the installation of pyhessio from the anaconda repository did not work. In fact it was better to git clone pyhessio and then python setup.py install. Which is what was corrected in #241 . I think the difference is that C library pyhessio files are compiled with setup.py install which is then compiled with the GCC from the user machine. Did not really check but it worked for me
